modules/phase_field/src/ics/BimodalInverseSuperellipsoidsIC.C

```diff
@@ -0,0 +1,164 @@
+/****************************************************************/
+/* MOOSE - Multiphysics Object Oriented Simulation Environment  */
+/*                                                              */
+/*          All contents are licensed under LGPL V2.1           */
+/*             See LICENSE for full restrictions                */
+/****************************************************************/
+
+#include "BimodalInverseSuperellipsoidsIC.h"
+#include "MooseMesh.h"
+
+template<>
+InputParameters validParams<BimodalInverseSuperellipsoidsIC>()
+{
+  InputParameters params = validParams<BimodalSuperellipsoidsIC>();
+  params.addClassDescription("Bimodal size distribution of large particles (specified in input file, value invalue) and small particles (placed randomly inside the larger particles, value outvalue)");
+  return params;
+}
+
+BimodalInverseSuperellipsoidsIC::BimodalInverseSuperellipsoidsIC(const InputParameters & parameters) :
+    BimodalSuperellipsoidsIC(parameters)
+{
+}
+
+void
+BimodalInverseSuperellipsoidsIC::initialSetup()
+{
+  switch (_size_variation_type)
+  {
+    case 2: //No variation
```

Comment:
Well to be honest only because `MultiSmoothCircleIC` did it the same way, and I was basing this code on that. Would it be better to replace it with an `if` condition instead?